pkg/cvo/cvo_scenarios_test.go

```diff
@@ -3937,6 +3938,10 @@ func verifyAllStatusOptionalDone(t *testing.T, ignoreDone bool, ch <-chan SyncWo
 			expect.Done = actual.Done
 		}
 
+		sort.Slice(actual.CapabilitiesStatus.ImplicitlyEnabledCaps, func(i, j int) bool {
+			return actual.CapabilitiesStatus.ImplicitlyEnabledCaps[i] < actual.CapabilitiesStatus.ImplicitlyEnabledCaps[j]
```

Comment:
can we sort these when we set `status`, where they'd also help with stability in live clusters, and not just here in the test-case?

Reply:
sure, I'll move it

Reply:
fixed at 54b69fcd6aba029fce086309662aa3b3c0b13386